### PR TITLE
Add weight service

### PR DESCRIPTION
**Pull request by kmaehashi**
This PR adds support Weight engine support (https://github.com/jubatus/jubatus/issues/1105).

Debugging TF-IDF weights for each feature is a common task; now jubakit can do that.

### CODE REVIEW

File: example/weight_shogun.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+"""
+Using Weight Service
+====================
+
+This exampel illustrates how to use Weight engine to debug fv_converter
```
Comment (rimms) [inline example/weight_shogun.py]: s/exampel/example

Comment (kmaehashi) [inline example/weight_shogun.py]: Fixed.

**Comment by rimms** [comment]
LGTM 👍

**Comment by kmaehashi** [comment]
Fixed missing line directly in 09f3577a526c3a58fc45473e7964932c2be6fde4

**Pull request merged**